### ISSUE TITLE
Feature request : allow unattented ssh connection until a command is finished

### ISSUE THREAD

**Issue by euidzero** [trustcrypto/onlykey-agent#34]
Trying to use onlykey-agent with ansible to manage multiple hosts in parallel or even serialy :
 the agent requires pin chalenge for each host,  thus making mass ssh impossible.

I suggess to add a feature to "unlock the ssh key" until a command has ended :

onlykey-agent identity@host --unlock-until-finished -- ansible-playbook play.yml
Enter the 3 digit challenge code on OnlyKey to authorize identity@host
..
ansible runs via  ssh on host1
ansible runs via ssh to host2
...

basically this means having to enter the challenge once per command, not per host.
